week-3/B-array-some/exercise.js

```diff
@@ -15,10 +15,11 @@ var pairsByIndex = [[0, 3], [1, 2], [2, 1], null, [3, 0]];
 var students = ["Islam", "Lesley", "Harun", "Rukmini"];
 var mentors = ["Daniel", "Irina", "Mozafar", "Luke"];
 
-var pairs = pairsByIndex.map(function(indexes) {
-  var student = students[indexes[0]];
-  var mentor = mentors[indexes[1]];
-  return [student, mentor];
+var containNull = pairsByIndex.some(function (include) {
+  include === null;
+  console.log(" null value is found in this array");
+  process.exit(1)
```

Comment:
So the purpose of this exercise was to **check** if the array contains a `null` value. If it does - call `process.exit(1)`. You can **check** something with a conditional - think about what the `some()` method returns and how you can use this.

Also, the instructions say - don't edit any of the existing code - just add your own check underneath `pairsByIndex` declaration.

Try again! :)

Reply:
I'll do.